### ISSUE TITLE
[Accessibility][Usable]: Narrator doesn't narrates the position number of element "Option d" without expanding it inside Basic uncontrolled example (allowFreeform: T, AutoComplete: T).

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11209]
Split out of issue #6920

**Comment by joschect** [comment]
Should look at the way that dropdown handles this because it does not reproduce this issue.

**Comment by xugao** [comment]
we need to make fix similar to this: https://github.com/OfficeDev/office-ui-fabric-react/pull/11374
perhaps exact something out that works for all scenarios: dropdown, comobox, menu

**Comment by micahgodbolt** [comment]
ok, looks like you can set aria-setsize, and aria-posinset on the input and it'll read it out without the dropdown being opened. I'll look at doing that tomorrow.

**Comment by xugao** [comment]
@micahgodbolt i spent a good amount of time looking into this today. simplying add `setsize` and `posinset` only work when you first time focus on the combobox. but if then you select next option by pressing down key (not expanding the menu options), it doesn't announce the position/size

I also tried always render the menu options but just leave it hidden. this way works better. however i don't think we can do this. we have `persistMenu` prop and it could be a breaking change if we just always render the callout/menu options.

Pls take a stub at fixing this :)

**Comment by xugao** [comment]
adding myself back as assignee since i want to pay close attention on this issue :)

**Comment by xugao** [comment]
Re-thinking this issue ... I don't see a reason why narrator should announce the position without expanding the dropdown. That is not an information sighted users can get neither.
